free-programming-books.md

```diff
@@ -1771,6 +1771,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Node.js Notes](http://books.goalkicker.com/NodeJSBook) (PDF)
 * [Node.js Succinctly, Syncfusion](https://www.syncfusion.com/resources/techportal/ebooks/nodejs) (PDF, Kindle) (email address *requested*, not required)
 * [The Node Beginner Book](http://nodebeginner.org)
+* [Reactive Programming with Node.js](http://www.allitebooks.com/reactive-programming-with-node-js/) - Fernando Doglio (PDF)
```

Comment:
Sorry, didn't notice the first time I looked!
Could you also keep the lists in alphabetical order?

R comes before T!